IndicoV2.Abstractions/Jobs/Models/IJobResult.cs

```diff
@@ -0,0 +1,9 @@
+﻿using System;
+
+namespace IndicoV2.Jobs.Models
+{
+    public interface IJobResult
+    {
+        Uri Url { get; }
+    }
```

Comment:
Jobs are long running tasks that can return `Json` or `JsonArray`. Although in certain circumstances a Job returns url (or urls), it's not always the case.

Reply:
I'll rename this to IWorkflowSubmissionJobResult, but first [I want to get some details about JobResult](https://indicodata.slack.com/archives/C01FMD4KJ92/p1611754959019400).

Reply:
Fixed